applications/luci-app-statistics/root/usr/libexec/stat-genconfig

```diff
@@ -32,36 +32,50 @@ const plugins = {
 	],
 };
 
-function parse_units(ustr) {
-	let val = 0;
+// only interval is using right now but let be all available for future use
+const load_plugin_schema = [
+	[ 'Interval', 'FlushInterval', 'FlushTimeout' ],
+	[ 'Globals' ],
+	[]
+];
 
+function parse_units(ustr) {
 	// unit map
 	const map = {
 		y  : 60 * 60 * 24 * 366,
-		m  : 60 * 60 * 24 * 31,
+		mon  : 60 * 60 * 24 * 31,
```

Comment:
This will not always hold true but I guess it's fine since it's already there. You've also redefined the behaviour for `m`, so I think m must be retained as the month calculation.